Bukkit/src/main/java/com/plotsquared/bukkit/listener/EntityEventListener.java

```diff
@@ -32,11 +32,7 @@
 import com.plotsquared.core.plot.Plot;
 import com.plotsquared.core.plot.PlotArea;
 import com.plotsquared.core.plot.PlotHandler;
-import com.plotsquared.core.plot.flag.implementations.DisablePhysicsFlag;
-import com.plotsquared.core.plot.flag.implementations.EntityChangeBlockFlag;
-import com.plotsquared.core.plot.flag.implementations.ExplosionFlag;
-import com.plotsquared.core.plot.flag.implementations.InvincibleFlag;
-import com.plotsquared.core.plot.flag.implementations.ProjectileChangeBlockFlag;
```

Comment:
Don't optimize imports, we don't use wildcard-imports. (https://github.com/IntellectualSites/PlotSquared/blob/main/.editorconfig#L70)

Reply:
Oh, okay. Thanks.